win32ss/gdi/ntgdi/freetype.c

```diff
@@ -3957,6 +3955,10 @@ TextIntGetTextExtentPoint(PDC dc,
     pmxWorldToDevice = DC_pmxWorldToDevice(dc);
     FtSetCoordinateTransform(face, pmxWorldToDevice);
 
+    FtMatrixFromMx(&mat, pmxWorldToDevice);
+    mat.yx = -mat.yx;
```

Comment:
Why are the following two lines needed?

Reply:
To be honest, I don't know why... (sigh)

See below code:
```c
    FtMatrixFromMx(&matWorld, pmxWorldToDevice);
    matWorld.yx = -matWorld.yx;
    matWorld.xy = -matWorld.xy;
```
If this code didn't exist, the glyph shape won't be matched.

Reply:
Ok. That's something to be thought about.